eventfinder/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(
+  <React.StrictMode>
```

Comment:
Feel free to keep strict mode in here, but be aware of what it does. I just recently encountered an issue with it and wasted time diagnosing why a component was unnecessarily re-rendering. It turned out to be StrictMode, which will intentionally re-render to help you diagnose caching issues and object reference changes.

We'll cover re-rendering and memoization at some point.

Reply:
I read up more on StrictMode. I decided to remove it temporarily, until covering re-rendering, memoization, and Concurrent React with you.